src/main/app/common/allowances/allowance.ts

```diff
@@ -1,35 +1,4 @@
-export class AllowanceItem {
-  item?: string
-  weekly?: number
-  monthly?: number
-
-  constructor (item?: string, weekly?: number, monthly?: number) {
-    this.item = item
-    this.weekly = weekly
-    this.monthly = monthly
-  }
-
-  static fromObject (value?: any): AllowanceItem {
```

Comment:
`fromObject` methods are typically used to reconstruct an instance from a form object. I am not quite sure if we need this and if we do we probably have a problem :-)